src/lib.rs

```diff
@@ -140,15 +143,18 @@ where
             write_opt,
             additionnal_tokens: (0, 0),
 
+            // For testing and debug purposes
             #[cfg(test)]
             blocking_duration: (Duration::ZERO, Duration::ZERO),
         }
     }
 
+    // Get the raw stream, deconstruct the Limiter struct.
```

Comment:
I think you need to place 3 / to be able to see it in the docs generated by `cargo doc`. Same for the others public functions above.